deploy/helm/metrics-collector-config.yaml

```diff
@@ -19,6 +19,20 @@ processors:
         statements:
           - convert_gauge_to_sum("cumulative", true) where IsMatch(metric.name, "^.*_total$") == true
 
+  # Remove datapoints of internal k8s containers from metrics, excluding the "container_network_.+_total" metric where other datapoints don't exist
```

Comment:
While `container_fs_*`, `container_spec_*`, ... contain datapoints both for normal containers and the internal containers, `container_network_*` contain only datapoints for the internal containers. So we cannot get rid of them in the beginning of the pipeline - instead, we can do it only after we used them to create other metrics.